src/operator/lrn.cc

```diff
@@ -40,6 +46,9 @@ Operator* CreateOp<cpu>(LRNParam param, int dtype) {
 #if MXNET_USE_MKL2017 == 1
   return new MKLLRNOp<cpu, float>(param);
 #endif
+#if MXNET_USE_MKLDNN == 1
+  return new MKLDNNLRNOp<cpu, float>(param);
```

Comment:
If both MKL2017 and MKLDNN are on, this would become dead code.

Reply:
As we do not support both flags at the same time, our MKLDNN build config will ensure MKL2017 is not be used.